packages/server/graphql/mutations/helpers/createGitLabTask.ts

```diff
@@ -0,0 +1,49 @@
+import {stateToMarkdown} from 'draft-js-export-markdown'
+import {GraphQLResolveInfo} from 'graphql'
+import splitDraftContent from 'parabol-client/utils/draftjs/splitDraftContent'
+import createIssueMutation from '../../nestedSchema/GitLab/mutations/createIssue.graphql'
+import {DataLoaderWorker, GQLContext} from '../../graphql'
+import GitLabServerManager from '../../../integrations/gitlab/GitLabServerManager'
+import {IGetTeamMemberIntegrationAuthQueryResult} from '../../../postgres/queries/generated/getTeamMemberIntegrationAuthQuery'
+import {CreateIssueMutation} from '../../../types/gitlabTypes'
+
+const createGitLabTask = async (
+  rawContent: string,
+  fullPath: string,
+  gitlabAuth: IGetTeamMemberIntegrationAuthQueryResult,
+  context: GQLContext,
+  info: GraphQLResolveInfo,
+  dataLoader: DataLoaderWorker
+) => {
+  const {accessToken, providerId} = gitlabAuth
+  if (!accessToken) return {error: new Error('Invalid GitLab auth')}
+  const {title, contentState} = splitDraftContent(rawContent)
+  const body = stateToMarkdown(contentState)
+  const provider = await dataLoader.get('integrationProviders').load(providerId)
```

Comment:
-1 prefer `loadNonNull` when the return value is guaranteed so we don't need `!` down below